gallery-so/gallery
# [GAL-3976] NFT selector support more chains

### Summary of Changes

- Added more chain in NFT selector chain filter
- Used the chain available from `shared` package
- Added a new util to get chain icon

### Demo or Before and After


https://github.com/gallery-so/gallery/assets/4480258/ef0c5689-418b-4285-9cdb-e39ab78d4678


### Edge Cases
- Filter each chain to make sure the filter logic works

### Testing Steps

1. Open nft selector
2. Press filter icon
3. You should able to see more chain in the modal

### Checklist

Please make sure to review and check all of the following:
